scriptmodules/ports/plex.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="plex"
+rp_module_desc="OpenPHT is a community driven fork of Plex Home Theater"
+rp_module_section="opt"
+rp_module_flags="!arm"
+
+VERSION="1.6.2.123-e23a7eef"
+APTVER=$(apt --version | cut -d' ' -f2) 
+DISTRO=$(lsb_release -is)
+CODENAME=$(lsb_release -cs)
+RELEASE=$(lsb_release -rs)
+PACKAGE="openpht_${VERSION}-${CODENAME}_amd64.deb"
+GETDEB="https://github.com/RasPlex/OpenPHT/releases/download/v${VERSION}/${PACKAGE}"
```

Comment:
Global vars are not allowed in modules - you can move them to the functions - use lowercase and local vars. You can use __raspbian_name / __raspbian_ver for comparing jessie / version.